src/assets/styles/GlobalStyles.scss

```diff
@@ -6,6 +6,12 @@
   }
 }
 
+@media (min-width: 1904px) {
+  .container {
```

Comment:
What is this guy for?

Reply:
if you are adding the max width for the whole view, please use it simularly as @media 
 wallet-content-container, and not container

Reply:
I understand what you mean, but I'm sure this is the right way to limit the web app width, because we use v-container in almost every components to limit their width.
There's no reason to create separate container class to limit the web app width.
<container> is made for this purpose.